new-client/src/plugins/BaseWindowPlugin.js

```diff
@@ -39,8 +39,8 @@ class BaseWindowPlugin extends React.PureComponent {
     this.title = props.options.title || props.custom.title;
     this.description = props.options.description || props.custom.description;
 
-    this.width = props.custom.width || 400;
-    this.height = props.custom.height || "auto";
+    this.width = props.options.width || 400;
+    this.height = props.options.height || "auto";
```

Comment:
This needs to be modified so that we a) use values from options if specified, b) or fall back to custom (custom per plugin, that is), or finally c) hard-coded values.